ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -385,7 +385,15 @@ objects:
                         --key /tmp/artifacts/installer/tls/journal-gatewayd.key \
                         --url "https://${bootstrap_ip}:19531/entries?_SYSTEMD_UNIT=${service}.service"
                 done
-              fi
```

Comment:
I think you need to keep a closing `fi` on `if [ -n "${bootstrap_ip}" ]`, although it should live after the block you add below, but before the `else` for `if [ -f /tmp/artifacts/installer/terraform.tfstate ]`.